packages/meditrak-server/src/dhis/DhisChangeValidator.js

```diff
@@ -110,9 +110,17 @@ export class DhisChangeValidator extends ChangeValidator {
     return this.queryValidSurveyResponseIds(surveyResponseIds);
   };
 
+  getValidEntityUpdates = async updateChanges => {
+    const entityIds = this.getIdsFromChangesForModel(updateChanges, this.models.entity);
+    if (entityIds.length === 0) return [];
+
+    const entities = await this.models.entity.findManyById(entityIds);
+    return entities.filter(e => e.allowsPushingToDhis()).map(e => e.id);
```

Comment:
This is where it actually filter the changes and prevent creating sync queue records for the updated entities that we don't want to push to DHIS2